.readthedocs.yml

```diff
@@ -21,7 +21,7 @@ build:
       - geovista download --operational --decompress
 
 conda:
-  environment: requirements/locks/rtd.yml
+  environment: requirements/geovista.yml
```

Comment:
This change is sub-optimal and increases the docs build time.

Overall, these changes allow the documentation to build on `RTD`, which was blocked due to an issue when attempting to `conda env create` an environment with `lazy_loader` only on `conda-forge`.

Ideally, we want to back this change out and use the pre-resolved `rtd.yml` for performance benefits on `RTD`.